__tests__/index.tsx

```diff
@@ -28,7 +28,7 @@ describe('hydration', () => {
     const result = readOutputFile('basic', 'index')
 
     // server renders correctly
-    expect(result).toMatch(
+    expect(result.replace(/(\r\n|\n|\r)/gm, '')).toContain(
```

Comment:
Rendered output now includes next js header scripts etc and various line breaks - so this test I've change to do a `contains` check instead.